project/src/test/java/com/google/sps/data/UserPrefrencesTest.java

```diff
@@ -0,0 +1,92 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import java.util.Arrays;
+import java.util.List;
+import com.google.maps.model.LatLng;
+import static org.junit.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@RunWith(JUnit4.class)
+public class UserPrefrencesTest {
+
+    private static final float RATING = 4;
+    private static final int PRICE_LEVEL = 2;
+    private static final LatLng LOCATION = new LatLng(32.08074, 34.78059);
+    private static final List<String> CUISINES = Arrays.asList("sushi", "burger");
```

Comment:
I think we should think about using an Enum or something of that sort for the different cuisines, because now when we start to use them we would want to be able to address them in a unified way.

Reply:
Absolutly. This regards the "add configuration file for multipe words textual search" task on our tasks list. I added an issue about it and will do it on a seperate PR on M1